python_modules/dagster/dagster/core/asset_defs/asset_group.py

```diff
@@ -508,9 +513,49 @@ def materialize(
             name="in_process_materialization_job", selection=selection
         ).execute_in_process()
 
+    def get_base_jobs(self) -> Sequence[JobDefinition]:
+        """For internal use only."""
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", category=ExperimentalWarning)
+
+            assets_by_partitions_def: Dict[
+                Optional[PartitionsDefinition], AssetsDefinition
```

Comment:
Should be `List[AssetsDefinition]` right?

Reply:
You are right